<ui version="4.0" >
 <class>BTPref</class>
 <widget class="QWidget" name="BTPref" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>269</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="dhtGroupBox" >
     <property name="title" >
      <string>DHT</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_dhtSupport" >
        <property name="toolTip" >
         <string>DHT is a trackerless protocol to find peers sharing the same torrents as you do.</string>
        </property>
        <property name="text" >
         <string>Use DHT to get additional peers</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <item>
         <widget class="QLabel" name="label_4" >
          <property name="text" >
           <string>UDP port for DHT communications:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="kcfg_dhtPort" >
          <property name="toolTip" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">UDP port to use for the DHT protocol.&lt;/p>
&lt;p style="-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600;">Attention:&lt;/span> If you are behind a router, this port needs to be forwarded to accept incoming DHT requests. The UPnP plugin can do this for you.&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="minimum" >
           <number>1</number>
          </property>
          <property name="maximum" >
           <number>65535</number>
          </property>
          <property name="value" >
           <number>4444</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_4" >
     <property name="title" >
      <string>Encryption</string>
     </property>
     <layout class="QVBoxLayout" >
      <item>
       <widget class="QCheckBox" name="kcfg_useEncryption" >
        <property name="toolTip" >
         <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Protocol encryption is useful when your ISP is slowing down bittorrent connections. &lt;/p>
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">The encryption will prevent your bittorrent traffic to be flagged as bittorrent traffic, and so the ISP will not slow it down.&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="text" >
         <string>Use protocol encryption</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="kcfg_allowUnencryptedConnections" >
        <property name="toolTip" >
         <string>Not all clients support encryption, and some people have encryption disabled. If you want to connect to those peers, you need to have this option enabled.</string>
        </property>
        <property name="text" >
         <string>Allow unencrypted connections</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kcfg_dhtSupport</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_dhtPort</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>355</x>
     <y>51</y>
    </hint>
    <hint type="destinationlabel" >
     <x>370</x>
     <y>75</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kcfg_useEncryption</sender>
   <signal>toggled(bool)</signal>
   <receiver>kcfg_allowUnencryptedConnections</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>113</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>116</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
